src/games/strategy/engine/framework/startup/mc/GameSelectorModel.java

```diff
@@ -380,11 +350,11 @@ public void run() {
         }).start();
       }
     } else {
-      if (!forceFactoryDefault) {
-        // we would rather have their game data refreshed after leaving a game
-        NewGameChooser.refreshNewGameChooserModel();
-      }
-      selectedGame = selectByName(ui, forceFactoryDefault);
+      // we would rather have their game data refreshed after leaving a game
+      NewGameChooser.refreshNewGameChooserModel();
+
+      String preferredMap = prefs.get(DEFAULT_GAME_NAME_PREF, DEFAULT_GAME_NAME);
+      selectedGame = selectByName(ui, preferredMap);
```

Comment:
Since the boolean variable forceFactoryDefault was removed, selectByName had an API change. selectByName needs that variable to know which property to use from preferences. Turns out, that we if pass in the property name, then we don't have to load up the preferences object multiple times (the other case in which we call this method we use the default game name, in which case we do not need to go to preferences)